spec/models/schema_contract/validation_initiator_spec.rb

```diff
@@ -77,5 +77,21 @@
         end.not_to raise_error
       end
     end
+
+    context 'when records exist that are over a month old' do
```

Comment:
I would suggest making two changes:
- I think this should go in spec/sidekiq/schema_contract/delete_validation_records_job_spec.rb. Doing this will unfortunately be another addition to the codeowners file. 
- rather than check for a change in the records count, I would explicitly check after the perform to ensure that the remaining record is the one we want.

Reply:
Good point. I was avoiding making a new file but you're right, it should be done properly. I made the changes